bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fullcalendar",
   "title": "FullCalendar",
-  "version": "2.3.2-bcherny",
+  "version": "2.3.2.1-bcherny",
```

Comment:
It's really just a unique identifier to differentiate between your version (`2.3.2-bcherny`) and this one.